diff --git a/core/CPM.cmake b/core/CPM.cmake
index c82a386..9a38ddd 100644
--- a/core/CPM.cmake
+++ b/core/CPM.cmake
@@ -709,6 +709,7 @@ function(CPMAddPackage)
   elseif(CPM_SOURCE_CACHE AND NOT CPM_ARGS_NO_CACHE)
     string(TOLOWER ${CPM_ARGS_NAME} lower_case_name)
     set(origin_parameters ${CPM_ARGS_UNPARSED_ARGUMENTS})
+    replace_abs_utility_to_rel(origin_parameters)
     list(SORT origin_parameters)
     if(CPM_USE_NAMED_CACHE_DIRECTORIES)
       string(SHA1 origin_hash "${origin_parameters};NEW_CACHE_STRUCTURE_TAG")
